### DISCUSSION TITLE
UI is not correctly loading in production

### DISCUSSION THREAD

**Discussion by paramwork** [filamentphp/filament#6433] [Help]
Please see the screenshot. The `/css/filament.css` file is loading but still, I am not seeing the correct UI design in the admin login and dashboard.

![chrome_NxkNiyUZL5](https://user-images.githubusercontent.com/63786245/236603649-3a99a0c9-98d6-48d4-b04a-19bb619cdf57.png)

**Comment by danharrin** [comment]
We cannot debug something like this for you. You need to provide more information, especially regarding console errors.

**Reply by paramwork** [reply]
I understand.

I am not getting any browser errors. Every function works as excepted only the layout is messed up. Some of the classes like "backdrop-blur-xl" and "filament-login-page" is not at all loading.

Do you know why it is happening and how I can debug it?

**Reply by danharrin** [reply]
I don't know why it is happening, its probably a bad server configuration. Using something official like Forge and Vapor works every time

**Reply by nicko1981** [reply]
I have exactly the same problem as this and am using Laravel Vapor.

The filament CSS file appears to be compiling differently in local and production (I have used npm run build on local to see what is compiled).

The CSS file on production comes in at 99.4Kb, where local version is 147Kb. 

As mentioned above, classes for filament-login-page, backdrop-blur-xl, text-3xl etc are missing from production version but present in local version.

I will note that my installation includes a custom colour, following the filament documentation exactly to do this, so the filament.css file being generated by Vite as an asset is the one that is built with the missing classes

**Comment by nicko1981** [comment]
I have discovered the problem in my installation was just a tiny mistake in the tailwind.config.js file, meaning filament blade views were not being included in the content section, so classes were being purged in production.

@paramwork suggest you check that './vendor/filament/**/*.blade.php', is being correctly listed in your tailwind config as a content path to be scanned for class inclusion.
